monorepo/website/src/pages/docs/how-to/authentication-api.mdx

```diff
@@ -12,7 +12,7 @@ The retrieved token expires after 10 hours.
 ## Send request with cURL
 
 <InfoBox>
-    To use the <a href="https://demo.pathoplexus.org" class="!text-white">demo instance</a> instead of the main instance, please replace `authentication.pathoplexus.org` with `authentication-demo.pathoplexus.org`.
+    To use the <a href="https://demo.pathoplexus.org" class="!text-black">demo instance</a> instead of the main instance, please replace `authentication.pathoplexus.org` with `authentication-demo.pathoplexus.org`.
```

Comment:
Should we maybe use a custom styled link component instead of applying the same class styling manually in 10 places? This predates the PR of course - just becomes noticeable due to the refactor in this PR.

Reply:
I don't think we should use a custom styled link component. If we made any change I think it should be about new CSS classes in the mdContainer's CSS

Reply:
I can also see an argument against mixing tailwind and custom CSS classes.

Another possible solution could be to add the styling to the infobox component and just style all links inside of it in a particular way, but that would also mix custom styling with tailwind.

Reply:
Example:

```astro
---
import MaterialSymbolsInfoOutline from '~icons/material-symbols/info-outline';
---

<style>
  .link-styled a {
    @apply text-red-500 underline; /* Tailwind classes */
  }

  .link-styled a:hover {
    @apply text-red-700; /* Tailwind classes for hover */
  }
</style>

<div class="bg-primary-300 p-3 my-4 rounded-lg mx-auto box text-sm link-styled">
  <span class="text-lg">
    <MaterialSymbolsInfoOutline />
  </span>
  <slot />
</div>

```

Reply:
I think this would be the easiest for the person writing the mdx file, you don't have to think about it at all when adding the link to the info box, just use the `a` tag without any stilying and it'll be fine.

Reply:
I like the idea of not putting custom styling in the mdx files - this should be in component code and transparent to docs writers.

I also don't see why we should use `<a>` at all in markdown - one should use `[link text](link target)` instead. @fhennig It'd be great if you could try out your suggestion, we could see whether turning the `<a>` into `<URL>` would work.

Reply:
it works! [ddf3c7d](https://github.com/pathoplexus/pathoplexus/pull/241/commits/ddf3c7dec3400bb5698188fddf9023abb1acc175)

Reply:
This is really cool, thanks @fhennig!
<img width="1571" alt="image" src="https://github.com/user-attachments/assets/7b050060-ad15-41a4-a4a0-42c5dfd2188b">

@theosanderson are you happy with this?